### PR TITLE
IPersonalAccountsApi - Refactor away from IApiResponse<T>

### PR DESCRIPTION
**Pull request by rayking29**
# Fixes [AB#14336](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/14336)

## Description

IPersonalAccountsApi - Refactor away from IApiResponse<T>
- removed IApiResponse and refactored code
- updated and removed unit tests
- updated method names for async naming standard

## Testing

- [x] Unit Tests Updated
- [ ] Functional Tests Updated
- [ ] Not Required

Screenshot 2022-11-22 at 1 31 27 PM

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by sslaws — APPROVED** [review]

File: Apps/Common/test/unit/Services/PersonalAccountServiceTests.cs [hunk 1]
```diff
@@ -82,31 +79,6 @@ public void ShouldGetPatientAccountUseCache()
             Assert.Equal(id, actualResult.ResourcePayload?.Id);
         }
 
-        /// <summary>
```
Comment (BrianMaki) [inline Apps/Common/test/unit/Services/PersonalAccountServiceTests.cs:85]: Where is the unit test to handle ApiException and HttpRequestException.  Won't his have on code coverage?

**Review by BrianMaki — CHANGES_REQUESTED** [review]

File: Apps/Common/test/unit/Services/PersonalAccountServiceTests.cs [hunk 2]
```diff
@@ -82,31 +79,6 @@ public void ShouldGetPatientAccountUseCache()
...
-        /// Get Personal Account returns ApiException.
-        /// </summary>
-        [Fact]
-        public void ShouldGetPatientAccountReturnsApiException()
```
Comment (BrianMaki) [inline Apps/Common/test/unit/Services/PersonalAccountServiceTests.cs:89]: Where is the unit test to handle ApiException?. Won't this have an effect on code coverage?  Looks like only HttpRequestException is being handled.

**Comment by rayking29** [comment]
I can't find this comment on my PR.

Either type of exception will go to the same exception code block so it won't have any effect on code coverage.
________________________________

File: Apps/Common/test/unit/Services/PersonalAccountServiceTests.cs (hunk 2 above)
Comment (rayking29) [reply Apps/Common/test/unit/Services/PersonalAccountServiceTests.cs:89]: See my previous comment. There shouldn't be any effect on code coverage. This is the same as the test changes that Stephen checked in earlier, one test for an exception.

**Review by BrianMaki — APPROVED** [review]

**Pull request merged**